OpenDreamClient/Rendering/DMISpriteComponent.cs

```diff
@@ -45,7 +45,8 @@ sealed class DMISpriteComponent : SharedDMISpriteComponent, ILookupWorldBox2Comp
                 if (!_entityManager.TryGetComponent<TransformComponent>(Owner, out var transform))
                     return false;
 
-                EntityUid mapEntity = IoCManager.Resolve<IMapManager>().GetMapEntityId(transform.MapID);
+                IoCManager.Resolve(ref mapManager);
```

Comment:
IoC dependency collections are backed by ThreadLocals which add up if you have to resolve it heaps, in ss14 we just add nullable arguments as an easy way around it if the class isn't ECS.